### PR TITLE
feat(css them): replace Bootstrap in the existing wc widgets

### PR DESCRIPTION
**Pull request by julienmege**
Closes [UID-548](https://bonitasoft.atlassian.net/browse/UID-548)

### CODE REVIEW

File: components/uid-input/stories/index.stories.ts [hunk 1]
```diff
@@ -58,3 +89,18 @@ SlottedContent.args = {
 SlottedContent.argTypes = {
   slot: { table: { disable: true } },
 };
+
+function getLabelHidden(labelHidden: boolean) {
+  if (!labelHidden) {
```
Comment (benjaminParisel) [inline components/uid-input/stories/index.stories.ts:94]: :x: didn't work like other boolean attributes

File: components/uid-text/src/uid-element.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
+import { LitElement, css, CSSResultGroup } from 'lit';
```
Comment (benjaminParisel) [inline components/uid-text/src/uid-element.ts:1]: :x: We really duplicate this class each time ? I think it's better to externalize it now (put a ref on npm repo, put this class in common folder and reference it when we need it)

File: components/uid-text/stories/index.stories.ts [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  component: 'uid-text',
+  argTypes: {
+    label: { control: 'text' },
+    labelHidden: { control: 'boolean' },
+    text: { control: 'text' },
+    labelPosition: {
+      options: ['top', 'left'],
+      control: 'radio',
+    },
+    labelWidth: { control: 'number' },
```
Comment (benjaminParisel) [inline components/uid-text/stories/index.stories.ts:15]: Where is allow-html ? :eyes:

File: components/uid-input/src/uid-input.ts [hunk 4]
```diff
@@ -1,3 +1,3 @@
-import { UidInput } from './UidInput.js';
+import { UidInput } from './UidInput';
```
Comment (benjaminParisel) [inline components/uid-input/src/uid-input.ts:1]: :question: Is it not possible to delete this file and put the code somewhere (in index.ts directly or in uidInput.ts). Same question for uid-text

File: components/uid-input/package.json [hunk 5]
```diff
@@ -49,39 +49,62 @@
     "@open-wc/testing": "next",
     "@typescript-eslint/eslint-plugin": "^4.31.2",
     "@typescript-eslint/parser": "^4.31.2",
-    "@web/dev-server": "^0.1.24",
-    "@web/dev-server-storybook": "^0.0.2",
+    "@web/dev-server": "^0.1.25",
+    "@web/dev-server-storybook": "next",
```
[the remark this answers is not in the document]
Comment (jeromecambon) [reply components/uid-input/package.json:53]: I agree, we should use the latest version number. "next" is a tag for latest version it seems.

File: components/uid-input/stories/index.stories.ts (hunk 1 above)
Comment (julienmege) [reply components/uid-input/stories/index.stories.ts:94]: will be done by @jeromecambon in another PR

File: components/uid-text/src/uid-element.ts (hunk 2 above)
Comment (julienmege) [reply components/uid-text/src/uid-element.ts:1]: In a next step we will extract this classe as a stand alone Npm module

**Pull request merged**